doc/dev/README.md

```diff
@@ -3,13 +3,15 @@
 A quick start guide to get Kepler up and running inside your container-based development cluster.
 
 ## Create a new emphemeral local kubernetes cluster
-Bring the cluster up with:
+Bring the cluster up with following [repo](https://github.com/sustainable-computing-io/KeplerK8SAction)
+
+## Deploying kepler in the cluster
```

Comment:
```suggestion
## Deploy Kepler to a local kind cluster
```

Reply:
en... maybe we should decouple CLUSTER_PROVIDER later? 
as `make cluster-sync` is able to deploy to any k8s cluster for now. but... we limited kind in current scripts. @rootfs , @marceloamaral what's your suggestion?

Reply:
I see what you mean, although the line below is `export CLUSTER_PROVIDER=kind` so that's why I recommended this change (so that users see the title and understand what's happening faster), but the current title is fine as well :) 👍